langs/python.go

```diff
@@ -75,8 +75,7 @@ func (h *PythonLangHelper) DockerfileBuildCmds() []string {
 	r = append(r, "ADD . /function/")
 	if exists("requirements.txt") {
 		r = append(r, `
-RUN apt-get update && apt-get install --no-install-recommends -qy build-essential gcc &&\
-    pip3 install --no-cache --no-cache-dir -r requirements.txt &&\
+RUN pip3 install --target /python/  --no-cache --no-cache-dir -r requirements.txt &&\
```

Comment:
I think you should add `--disable-pip-version-check` here  as it drops the spam and speeds up pip a tiny bit